docs/inventory/instance.rst

```diff
@@ -58,7 +58,7 @@ Parameters
 
 
   **strict (type=bool):**
```

Comment:
@yec-akamai When you get a chance could this file's diff be removed from this PR to fix the docs validation workflow?

Just for context, this is a result of the latest version of Ansible not supporting our minimum Python version but having some minor documentation differences. I think we're planning on bumping our minimum Python version in the future 👍 

See #625 for a bit more context